wfe2/wfe_test.go

```diff
@@ -2657,9 +2657,12 @@ func TestFinalizeOrder(t *testing.T) {
 			ExpectedBody: `{"type":"` + probs.V2ErrorNS + `orderNotReady","detail":"Order's status (\"pending\") is not acceptable for finalization","status":403}`,
 		},
 		{
-			Name:            "Good CSR, Ready Order",
-			Request:         signAndPost(signer, "1/8", "http://localhost/1/8", goodCertCSRPayload),
-			ExpectedHeaders: map[string]string{"Location": "http://localhost/acme/order/1/8"},
+			Name:    "Good CSR, Ready Order",
+			Request: signAndPost(signer, "1/8", "http://localhost/1/8", goodCertCSRPayload),
+			ExpectedHeaders: map[string]string{
+				"Location":    "http://localhost/acme/order/1/8",
+				"Retry-After": "3",
```

Comment:
This `Retry-After` header should not exist on an order in the ready state, only for an order in the processing state. The test still passes with this header being set, but I think that's a different issue.

Reply:
The order _was_ ready, but since this is `TestFinalizeOrder`, once the finalize request goes through the order is now processing (hence why the `ExpectedBody` below has `"status": "processing"`).

Reply:
I don't know how I missed that.